boards/x86/acrn/acrn_defconfig

```diff
@@ -0,0 +1,16 @@
+CONFIG_X86=y
+CONFIG_SOC_IA32=y
+CONFIG_BOARD_ACRN=y
+CONFIG_CPU_MINUTEIA=y
```

Comment:
We're not using a quark CPU.
Probably want to set CONFIG_CPU_APOLLO_LAKE instead.

Reply:
actually shouldn't the SOC set this? maybe just omit all together.

Reply:
MINUTEIA is set for every SOC_IA32, so it's an artifact of that, not a deliberate choice.
I'll pull it from the defconfig and regression test.

Reply:
probably an artifact of when we were only supporting Quark CPUs